src/test/resources/fixtures/hmh-assessment-service/caliperEventSpanishRiGraded.json

```diff
@@ -0,0 +1,49 @@
+{
+  "@context": "http://purl.imsglobal.org/ctx/caliper/v1p1",
+  "id":"urn:uuid:a50ca17f-5971-47bb-8fca-4e6e6879001d",  // Event Id
+  "type": "GradeEvent",
+  "actor":{
+    "id":"urn:uuid:0f4fedbe-2227-415f-8553-40731a627171", //Student Id
+    "type":"Person"
+  },
+  "action": "Graded",
+  "object": {
+    "id": "urn:uuid:1050e852-5edb-4743-92d1-b53466de3a5f", // last attempt id
+    "type": "Attempt",
+    "assignee": "urn:uuid:0f4fedbe-2227-415f-8553-40731a627171",  //student id
+    "assignable": {
+      "id":"urn:uuid:c050e852-5edb-4743-92d1-b53466de3a5f",
+      "type": "Assessment"
+    },
+    "count": 33,
+    "dateCreated": "2016-11-15T10:05:00.000Z",
+    "startedAtTime": "2016-11-15T10:05:00.000Z",
+    "endedAtTime": "2016-11-15T10:55:12.000Z"
+  },
+  "eventTime": "2016-11-15T10:57:06.000Z",
+  "edApp": "https://www.hmhco.com/AssessmentService",
```

Comment:
Another Assessment Service exists in HMH - just wondering is there a more specific name for your application by any chance?